src/service/chat.js

```diff
@@ -271,6 +271,53 @@ const initChatFromPostAnon = async ({source, id}) => {
   }
 };
 
+const addMemberGroup = async ({channelId, memberIds}) => {
+  try {
+    const response = await api.post('/chat/group/add-members', {
+      channel_id: channelId,
+      members: memberIds
+    });
+    if (response.status === 200) {
+      return Promise.resolve(response.data);
+    }
+    return Promise.reject(response.data);
+  } catch (e) {
+    if (e?.response?.data?.message) return Promise.reject(e?.response?.data?.message);
+    return Promise.reject(e);
+  }
+};
+
+const removeMemberGroup = async ({channelId, targetUserId}) => {
+  try {
+    const response = await api.post('/chat/group/remove-member', {
+      channelId,
+      targetUserId
+    });
+    if (response.status === 200) {
+      return Promise.resolve(response.data);
+    }
+    return Promise.reject(response.data);
+  } catch (e) {
+    if (e?.response?.data?.message) return Promise.reject(e?.response?.data?.message);
+    return Promise.reject(e);
+  }
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a mismatch in the request body of the `removeMemberGroup` function. The keys in the request body should match the expected keys on the server-side. If the server expects `channel_id` and `target_user_id`, then the request body should contain these keys instead of `channelId` and `targetUserId`.
```diff
-    const response = await api.post('/chat/group/remove-member', {
-      channelId,
-      targetUserId
-    });
+    const response = await api.post('/chat/group/remove-member', {
+      channel_id: channelId,
+      target_user_id: targetUserId
+    });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->